personal_tokens.yaml

```diff
@@ -0,0 +1,89 @@
+openapi: 3.0.3
+info:
+  title: Personal Tokens - OpenAPI
+  description: |
+    The Personal Tokens API allows you to manage your existing personal tokens.
+  license:
+    name: MIT
+    url: https://github.com/aziontech/azionapi-openapi/blob/main/LICENSE.md
+  version: 1.0.0
+externalDocs:
+  description: See the documentation on Personal Tokens for more information.
+  url: https://www.azion.com/en/documentation/products/accounts/personal-tokens/
+servers:
+  - description: Production
+    url: https://api.azionapi.net
+security:
+  - tokenAuth: []
+tags:
+  - name: Personal Token
+    description: List all personal tokens
+paths:
+  /iam/personal_tokens:
+    get:
+      tags:
+        - Personal Token
+      summary: List all existing personal token 
+      description: List all existing personal token
+      operationId: ListPersonalToken
+      
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PersonalTokenResponseWithResults'          
+components:
+  schemas:
+    PersonalTokenResponseWithResults:
+      type: object
+      properties:
+        count:
+          type: integer
+          format: int64
+        total_pages:
+          type: integer
+          format: int6
+        schema_version: 
+          type: integer
+          format: int64
+        links:
+          type: object
+          properties:
+            next:
+              type: string
+              nullable: true
+            previous:
+              type: string
+              nullable: true
+        results:
+            type: array
+            items:
+              $ref: '#/components/schemas/PersonalTokenResponseGet'
+  
```

Comment:
We don't describe any xml as return value in any other openapi specification, so remove this key _xml_ from here

Reply:
Done!https://github.com/aziontech/azionapi-openapi/pull/65/commits/1820bed0618b37d6dc612e687613cb918d4880d2